src/addon.ts

```diff
@@ -39,10 +38,10 @@ export async function createReleaseFromArchive(
 
     // create release
     const subFileBuffer = await file.arrayBuffer()
-    return createReleaseFromDll(addon, subFileBuffer, id, downloadUrl)
+    return createReleaseFromDll(subFileBuffer, id, downloadUrl)
   }
 
-  return undefined
+  throw new Error(`no valid release assets found in archive`)
```

Comment:
this could include the download url to make up for the missing addon name. Also applies to all the other error messages changed

Reply:
Fixed in #49